types/ext/settings.d.ts

```diff
@@ -371,9 +371,9 @@ export type PopupVerticalTextPosition = 'default' | 'before' | 'after' | 'left'
 
 export type GlossaryLayoutMode = 'default' | 'compact';
 
-export type PopupTheme = 'light' | 'dark' | 'browser';
```

Comment:
Was this just missed? It seemed to be missing so I just added it for completeness.